Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py

```diff
@@ -580,13 +580,40 @@ def test_search_records(self, requests_mock):
         assert records[0]['record']['Id'] == '238756'
         assert records[0]['record']['Device Name'] == 'DEVICE NAME'
 
-    @pytest.mark.parametrize('field_name,field_to_search_by_id,expected_condition', [
-        ('id_field_name', '', '<TextFilterCondition>        <Operator>Contains</Operator>        '
-         + '<Field name="id_field_name">field_id</Field>        <Value>1234</Value></TextFilterCondition >'),
-        ('id_field_name', 'id_field_name', '<ContentFilterCondition>        <Level>5678</Level>        '
-         + '<Operator>Equals</Operator>        <Values><Value>1234</Value></Values></ContentFilterCondition>')
-    ])
-    def test_search_records_soap_request(self, field_name, field_to_search_by_id, expected_condition):
+    @pytest.mark.parametrize(
+        'field_name,field_to_search_by_id,expected_condition',
+        [
+            pytest.param(
+                'id_field_name',
+                '',
+                # Expected
```

Comment:
what's expected?

Reply:
I have improved the formatting in commit ecb64e0cf12b0a04d919aab713430ba092c3b336 to make it clearer

Unfortunately, `pytest.param` doesn't support kwargs. Here are a few ideas:
* Using a list of `collections.namedtuple` objects instead inside `@pytest.mark.parametrize`
* Breaking it up into multiple unit tests (improves readability but may make code less concise)

Let me know if you are happy with the reformatting or if you would like me to implement one of the two other options.